sys/riscv/include/machdep.h

```diff
@@ -42,6 +42,8 @@ struct riscv_bootparams {
 	vm_offset_t	kern_phys;	/* Kernel base (physical) addr */
 	vm_offset_t	kern_stack;
 	vm_offset_t	dtbp_virt;	/* Device tree blob virtual addr */
+	vm_offset_t	dtbp_phys;	/* Device tree blob physical addr */
+	vm_size_t	dtbp_size;	/* Device tree blob size */
```

Comment:
This doesn't belong here, this struct is purely for communicating information from locore to C.